libs/entry-components/styles/modules/_default-theme.scss

```diff
@@ -61,7 +61,8 @@ $theme: (
 			selected-color: #FFF,
 			disabled-color: #F5F5F5,
 			odd-even-row: odd,
-			odd-even-background: #F0F0F0
+			odd-even-background: #F0F0F0,
+			odd-even-background-selected: null
```

Comment:
This is actually selected-background? Odd-even has less to do with it.

Reply:
I renamed it to selected-background and to avoid confusion with selected color, I decided to nest them together.